report/2.Analysis/TrackingDetectionLocalization.tex

```diff
@@ -0,0 +1,19 @@
+\section{Object detection, localization and tracking}
+Before examining the different ways to look for objects using a computer, it is important to ensure the correct understanding of the terminology used when working with objects in images.
+Object recognition is the discipline of computers working with objects in images, and object detection, localization and tracking, are all subtasks of object recognition.
+The difference between detection, localization and tracking, is subtle, however it is necessary to understand the difference, as it is quite important\cite{objecttrackdetect}.
+
+Object localization, is the matter of determining the location of a single object in an image. 
+Here it is irrelevant whether it is a bounding box for the object, an actual outline, or just an average location.
+
+Object detection is the matter of determining the locations of multiple objects in an image.
+Usually the location of each individual object will be outlined with a bounding box, to ensure clarity as to which pixels represent each object.
+
+Object tracking is the matter of doing either object localization or object detection over time and storing the previous locations.
+
+Another aspect of Computer Vision, is object classification, or determining the class, or type, of one or more objects in an image.
```

Comment:
hvorfor står computer vision med stort??????????????

Reply:
Object classification should be the very first thing, since it is the simplest